src/test/java/org/broadinstitute/consent/http/resources/ElectionResourceTest.java

```diff
@@ -32,7 +31,6 @@
 @SuppressWarnings("FieldCanBeLocal")
 @RunWith(PowerMockRunner.class)
 @PowerMockIgnore("jdk.internal.reflect.*")
-@PrepareForTest({AbstractElectionAPI.class})
```

Comment:
I think we can get rid of all of these if we aren't using `AbstractElectionAPI` any more